Machine_Learning/Student_Burnout_Detector/app.py

```
@@ -0,0 +1,52 @@
+activity = st.slider("Physical activity (hours)", 0, 5, 1)
+mood = st.slider("Mood level (1=low, 5=high)", 1, 5, 3)
+assignments = st.slider("Number of assignments", 0, 10, 2)
+caffeine = st.slider("Caffeine intake (cups)", 0, 5, 1)
+social = st.slider("Social interaction (hours)", 0, 10, 2)
+
+if st.button("🔍 Predict Burnout Risk"):
+   # Match feature order used during training: sleep_hours, screen_time, physical_activity, mood_level, assignments, caffeine, social_interaction
+    feature_names = ['sleep_hours', 'screen_time', 'physical_activity', 'mood_level', 'assignments', ' caffeine', 'social_interaction']
+    user_data = np.array([[sleep, screen, activity, mood, assignments, caffeine, social]])
+        # Validate feature count
+    if user_data.shape[1] != len(feature_names):        st.error(f"Feature count mismatch: expected {len   (feature_names)}, got {user_data.shape[1]}")
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)